src/Showpad/Client.php

```diff
@@ -163,13 +163,32 @@ public function assetsTagsAddById($id, $tag)
 
         // Create request
         $data = $this->auth->request(
-            'GET',
+            'LINK',
```

Comment:
Seems not.
Just tested with GET on a working app and it don't work. (But no errors in logs also)
With LINK this method works as expected.